src/components/ChatApp/Settings/PreviewThemeChat.js

```diff
@@ -3,43 +3,79 @@ import PropTypes from 'prop-types';
 import { getSusiPreviewReply } from '../../../apis/index';
 import susiWhite from '../../../images/susi-logo-white.png';
 import MoreVertIcon from 'material-ui/svg-icons/navigation/more-vert';
+import loadingGIF from '../../../images/loading.gif';
 import Send from 'material-ui/svg-icons/content/send';
 import './PreviewThemeChat.css';
 
 class PreviewThemeChat extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      messages: [{ message: 'Hi I am Susi', author: 'Susi' }],
+      messages: [{ message: 'Hi, I am SUSI', author: 'SUSI', loading: false }],
       message: '',
     };
   }
 
   sendMessage = event => {
-    event.preventDefault();
-    this.addMessage(this.state.message, 'You');
+    if (this.state.message.trim().length > 0) {
+      this.addMessage(this.state.message, 'You');
 
-    let message = encodeURIComponent(this.state.message);
-    getSusiPreviewReply(message)
-      .then(payload => {
-        if (payload.answers[0]) {
-          this.addMessage(payload.answers[0].actions[0].expression, 'SUSI');
-        } else {
-          this.addMessage(
-            'Sorry, I could not understand what you just said.',
-            'SUSI',
-          );
-        }
-      })
-      .catch(error => {
-        console.log('Could not fetch reply');
-      });
-    this.setState({ message: '' });
+      let message = encodeURIComponent(this.state.message);
```

Comment:
Rename variable to some other name, and I see destructuring clearly working out.